src/main/java/org/broadinstitute/hellbender/utils/Nucleotide.java

```diff
@@ -184,12 +274,37 @@ public static Nucleotide decode(final byte base) {
 
     /**
      * Returns the nucleotide that corresponds to a particular {@code char} typed base code.
-     * @param base the query base code.
+     * @param ch the query base code.
      * @return never {@code null}, but {@link #INVALID} if the base code does not correspond
      * to a valid nucleotide specification.
      */
-    public static Nucleotide decode(final char base) {
-        return baseToValue[base & 0xFF];
+    public static Nucleotide decode(final char ch) {
+        if ((ch & 0xFF00) != 0) {
+            return INVALID;
+        } else {
+            return baseToValue[ch & 0xFF];
+        }
+    }
+
+    /**
+     * Transform a single-letter character string into the corresponding nucleotide.
+     * <p>
+     *    {@code Null}, empty or multi-letter input will result in an {@link IllegalArgumentException}.
+     *    These are not simply invalid encodings as the fact that are not a single character is
```

Comment:
encodings as the fact that are not a single character  -> encodings, and the fact that they are not a single character?